packages/api/cloud/src/auth.rs

```diff
@@ -271,24 +277,26 @@ impl Auth {
 			let (_, user_ent) = self.user(ctx).await?;
 
 			// Fetch teams associated with user
-			let teams_res = op!([ctx] user_team_list {
-				user_ids: vec![user_ent.user_id.into()],
-			})
+			let teams_res = chirp_workflow::compat::op(
+				&ctx,
+				::user::ops::team_list::Input {
+					user_ids: vec![user_ent.user_id.into()],
+				},
+			)
 			.await?;
 			let user = unwrap!(teams_res.users.first());
-			let team_ids_proto = user
+			let team_ids = user
 				.teams
 				.iter()
-				.filter_map(|t| t.team_id)
-				.collect::<Vec<common::Uuid>>();
-			let team_ids = team_ids_proto
-				.iter()
-				.map(common::Uuid::as_uuid)
+				.map(|t| t.team_id)
 				.collect::<Vec<_>>();
 
 			// Fetch games associated with teams
 			let games_res = op!([ctx] game_list_for_team {
-				team_ids: team_ids_proto,
+				team_ids: team_ids
+					.iter()
+					.map(|id| (*id).into())
+					.collect::<Vec<common::Uuid>>()
```

Comment:
Nit: you can leave out the type in `.collect` statements if there is enough type info for the compiler to predict the type:
```suggestion
					.collect::<Vec<_>>()
```
or even
```suggestion
					.collect::<_>()
```